package.json

```diff
@@ -68,7 +71,7 @@
     "stylelint-scss": "3.18.0",
     "ts-jest": "^26.5.6",
     "ts-node": "^10.9.1",
-    "typescript": "4.2.4",
+    "typescript": "4.8.4",
```

Comment:
did you check if production build (`yarn build`) still works after this change?

Reply:
Hmm right, there is the error. Generally solution everywhere is to bump to latest version `ts-node` and `typescript` but in this case error occurs. I downgraded version of `typescript` to `4.6.4` and it works fine. It doesn't make much difference to us, we'll wait until they fix the latest versions of both libraries.